deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -610,8 +611,9 @@ def _gen_message_for_refreshing_cluster(request, last_cluster_refresh_status, la
 
 def _getLastClusterRefreshStatus(request, env):
     try:
+        cluster_name = get_cluster_name(request, env.name, env.stage)
         replace_summaries = clusters_helper.get_cluster_replacement_status(
-            request, data={"clusterName": '{}-{}'.format(env["envName"], env["stageName"])})
+            request, data={"clusterName": '{}'.format(cluster_name)})
```

Comment:
nit: we can replace this with
```suggestion
            request, data={"clusterName": 'cluster_name})
```

Reply:
There is a typo, no single quote needed ^^